src/file-base.js

```diff
@@ -229,6 +255,9 @@ export function computeHash(fileStream, fileSize, algorithm, progress) {
         if (typeof progress === 'function') {
           progress(100)
         }
+        if (cache && file != null) {
+          file._computedHashes[algorithm] = hash
+        }
```

Comment:
`computedHash()` funciton is still coupled with File class. My suggestion erlier was to not use any cache related thing in this function. Do this only in the main `hash()` function. I think `computedHash()` should be away from the cache related stuffs, as it should do only one job.

Reply:
agree with @kmanaseryan 100%.

Reply:
Fixed! Thanks

Reply:
@risenW I still see this, did miss something?

Reply:
This is the old commit. Review the updated one here: https://github.com/frictionlessdata/frictionless-js/pull/111/commits/3c30fb550cbdc9dea2f80c73462871fd93e8b981